checks/core/deployed_version.py

```diff
@@ -4,20 +4,57 @@
 The deployed version and the latest tag are returned.
 """
 from poucave.typings import CheckResult
-from poucave.utils import fetch_json
+from poucave.utils import ClientSession, fetch_json, retry_decorator
 
 EXPOSED_PARAMETERS = ["server", "repo"]
 
 
-async def run(server: str, repo: str) -> CheckResult:
-    # Server __version__ (see mozilla-services/Dockerflow)
-    version_info = await fetch_json(server + "/__version__")
+@retry_decorator
+async def fetch_latest_tag(owner: str, name: str, token: str) -> str:
+    url = "https://api.github.com/graphql"
+    data = {
+        "query": """
+            query {
+              repository(owner: "%s", name: "%s") {
+                refs(refPrefix: "refs/tags/", last: 1, orderBy: {field: TAG_COMMIT_DATE, direction: ASC}) {
+                  edges {
+                    node { name }
+                  }
+                }
+              }
+            }
+        """
+        % (owner, name)
```

Comment:
Do we have to quote-escape these or something? I guess it's probably fine because we expect these values to come from configuration..

Reply:
Yeah, I wouldn't mind if poucave fails with quotes in owner/project strings :)